bin/git-bulk

```diff
@@ -36,7 +36,11 @@ function addworkspace {
             source=$(realpath "$source" 2>/dev/null)
             if [ -f "$source" ]; then
                 pushd "$wsdir" > /dev/null
-                while read -r line; do git clone "$line"; done < "$source";
+                while IFS= read -r line || [[ -n $line ]]; do
+                  if [ -n "$line" ]; then
+                    git clone $line;
```

Comment:
Let‘s add a comment that it's dedicated not to quoting the var.

Reply:
I did not get you. Could you please elaborate?
Thank you.

Reply:
It's intended not to use `"$line"` here, right?

Reply:
If double quotes are used, the resulting git clone command will be:

```
git clone "https://host-of-git/repo-1.git Repo-1-Code"
```

This command may result in an invalid URL since the repository URL and the destination folder are enclosed in double quotes.

However, if the double quotes are removed, the git clone command will be:

```
git clone https://host-of-git/repo-1.git Repo-1-Code
```

By removing the double quotes, the command will create a separate folder named "Repo-1-Code" for the cloned repository, which is the enhancement I am suggesting.

Reply:
@icodebuster 
Would you add the explanation as a comment in the code?

Reply:
I have added a short comment as an explanation. If you have any alternate comments/suggestions, do let me know.